rest_api/operator_apis/dns-operator-openshift-io-v1.adoc

```diff
@@ -266,20 +271,20 @@ Required::
 
 | `clusterDomain`
 | `string`
-| clusterDomain is the local cluster DNS domain suffix for DNS services. This will be a subdomain as defined in RFC 1034, section 3.5: https://tools.ietf.org/html/rfc1034#section-3.5 Example: "cluster.local" 
+| clusterDomain is the local cluster DNS domain suffix for DNS services. This will be a subdomain as defined in RFC 1034, section 3.5: https://tools.ietf.org/html/rfc1034#section-3.5 Example: "cluster.local"
```

Comment:
Use present tense: "This is a subdomain..."

Reply:
I see so many instances of future tense  in the existing content. Let me fix that. Good catch.